### ISSUE TITLE
"You were banned for hacking"

### ISSUE THREAD

**Issue by fffelix-jan** [CartoonChess/among-us-mac#5]
When playing with a freshly-built copy, it suddenly stops with the message, "You were banned for hacking. Please stop."

**Comment by CartoonChess** [comment]
Thanks for the report. I've just tested on v2020.9.22i (latest from itch.io) with no problems. What is the source for your game data? Do you have any game trainers or memory readers that might trigger this warning?

According to the Innersloth Discord, they've added some anti-hacking protections recently, but I couldn't find any specific details. However, some posts on Reddit suggest that if one person in a lobby is using any hacks, everyone is booted with the same message. Haven't seen it myself so can't confirm.

Can you try a couple more games and see if the message persists? Thank you.

**Comment by fffelix-jan** [comment]
No, I don't use such tools. I think it could just be lag. I am on a 2014 Mac mini, and the framerate seems to be choppy throughout. Later on, that message stopped appearing, but I think it could just be lag.

**Comment by CartoonChess** [comment]
I believe you’re most likely right with the issue being the lag. I have seen reports in passing that users of the official client face a similar issue.

Although the official system requirements aren’t very strict, this Mac version presumably uses more resources; in testing on a recent MacBook Pro, the fans really whine.

I’ll make a note in the readme about possible system requirements and lag. Thanks for testing!

Oh, and if you’ve got time, could you check issue #3 and see if you experienced something similar? The problem is in need of feedback. Thanks 😅
